http/scraper_service_test.go

```diff
@@ -675,7 +683,9 @@ func TestService_handlePatchScraperTarget(t *testing.T) {
 		              "links":{
 		                "bucket": "/api/v2/buckets/0000000000000212",
 		                "organization": "/api/v2/orgs/0000000000000211",
-		                "self":"/api/v2/scrapers/%[1]s"
+		                "self":"/api/v2/scrapers/%[1]s",
```

Comment:
These IDs look odd; would you check why they aren't `targetOneID` ?

Reply:
```
body: fmt.Sprintf(`..., "self": "/api/v2/scrapers/%[1]s",...`, targetOneIDString) 
```

Reply:
The `%[1]s` is to maintain the same code style as the previous code.
I think It is clearer to remove `fmt.Sprintf` and replace   `%[1]s` with `targetOneID`.

Reply:
Ok. would you remove the [1] from the string?  That looks like a bug to me.

Reply:
Actually, that is not a bug. Just different code styles.
```
fmt.Sprintf(
	`
{
  "id": "%[1]s",
  "name": "name",
  "type": "prometheus",
  "url": "www.example.url",
  "organization": "org1",
  "orgID": "0000000000000211",
  "bucket": "bucket1",
  "bucketID": "0000000000000212",
  "links": {
    "bucket": "/api/v2/buckets/0000000000000212",
    "organization": "/api/v2/orgs/0000000000000211",
    "self": "/api/v2/scrapers/%[1]s",
    "members": "/api/v2/scrapers/%[1]s/members",
    "owners": "/api/v2/scrapers/%[1]s/owners"
  }
}`,
	targetOneIDString,
)
```
All `%[1]s` will be replace with `targetOneIDString`
(https://golang.org/pkg/fmt  #Explicit argument indexes)

If remove `[1]` from the string, that will like:
```
fmt.Sprintf(
	`
{
  "id": "%s",
  "name": "name",
  "type": "prometheus",
  "url": "www.example.url",
  "organization": "org1",
  "orgID": "0000000000000211",
  "bucket": "bucket1",
  "bucketID": "0000000000000212",
  "links": {
    "bucket": "/api/v2/buckets/0000000000000212",
    "organization": "/api/v2/orgs/0000000000000211",
    "self": "/api/v2/scrapers/%s",
    "members": "/api/v2/scrapers/%s/members",
    "owners": "/api/v2/scrapers/%s/owners"
  }
}`,
	targetOneIDString, targetOneIDString, targetOneIDString, targetOneIDString
)
```
Which style do you prefer?  @goller

Reply:
`[1]` removed.

Reply:
Not @goller is not a bug. It was intended to use only one parameter in the `fmt.Sprintf` tail rather than repeting the same variable 3 times (I was the author of this choice months ago).

Reply:
Personally I prefer the original one.

Just because I do not like function calls (and also definitions) with so many parameters ...

Reply:
BTW, since I was the only to use this style and all the other APIs impl. do not use it I'm okay with conforming to the common style the team is using.